### DISCUSSION TITLE
v3.2.0

### DISCUSSION THREAD

**Discussion by coolapso** [streamdeck-linux-gui/streamdeck-linux-gui#111] [Announcements]
V3.2.0 has now been released.

You can find the release notes here: https://github.com/streamdeck-linux-gui/streamdeck-linux-gui/releases/tag/v3.2.0.

Flatpaks and Snap packages are still under investigation. If you have experience with such packaging systems and wish to submit a PR for the build, please feel free to do so. The same applies to any other packaging system that can make it easier for users to install the application.

V4.0.0 will be released soon, and there will be no beta version for it. V4.0.0 will introduce breaking changes to support highly requested features, such as multi-state buttons. The development cycle will also change. After v4.0.0, there will no longer be separate development and beta branches; everything will be integrated into the main branch. 

Thank you to all the contributors!

**Comment by oparada1988** [comment]
what is the correct command to update the package?

**Reply by coolapso** [reply]
if installed via pip: `pip install [package_name] --upgrade`
if installed via source, repeat the process.

**Comment by oparada1988** [comment]
I thought so, on Ubuntu 22.04 it keeps installing an older version, even after removing 3.1.0 it just goes back to the previous project:

![Screenshot from 2023-10-29 17-11-23](https://github.com/streamdeck-linux-gui/streamdeck-linux-gui/assets/4031078/e8f6cac4-2551-4256-9ec6-1a9677931a9e)

![Screenshot from 2023-10-29 17-11-54](https://github.com/streamdeck-linux-gui/streamdeck-linux-gui/assets/4031078/85aa205b-1d20-4fb6-9e2d-5f94fd6f4eae)

**Reply by coolapso** [reply]
you are missing the `--upgrade` flag

**Comment by oparada1988** [comment]
I do a clean install and it still stuck in 3.0.1

![Screenshot from 2023-10-29 19-04-44](https://github.com/streamdeck-linux-gui/streamdeck-linux-gui/assets/4031078/b5d4a128-86e3-471b-b0dc-69382860ec93)

![Screenshot from 2023-10-29 19-05-06](https://github.com/streamdeck-linux-gui/streamdeck-linux-gui/assets/4031078/dfc388fc-d9d4-416f-9281-be13c8653e5e)

![Screenshot from 2023-10-29 19-05-36](https://github.com/streamdeck-linux-gui/streamdeck-linux-gui/assets/4031078/39cbad79-784a-441b-af1d-7f1fc1717dde)

Hopefully the flatpak version is around the corner...

**Reply by coolapso** [reply]
> Hopefully the flatpak version is around the corner... 

not really ... they are not as easy to make as one thinks!

**Comment by oparada1988** [comment]
Is it related to #81?

**Reply by coolapso** [reply]
no. you are using the right package. what is your python version?

**Reply by oparada1988** [reply]
Hey, sorry about the late reply( got a one month old at home) I am runing 3.10.12

![Screenshot from 2023-10-30 19-14-30](https://github.com/streamdeck-linux-gui/streamdeck-linux-gui/assets/4031078/a4aa73fe-f04a-4c1e-9f96-872237d325e8)

**Reply by coolapso** [reply]
hmmm ... Can you try updating to python 3.11 and see if it changes? I think there was a change on the python version and maybe you are getting only the one that is supported by your python version

**Reply by coolapso** [reply]
yeah almost sure that is it ... since v3.1 python 3.11 is required

**Reply by coolapso** [reply]
@rkmax  do we really require python 3.11 or can we tune it down for > 3.8 as a requirement for the package? :thinking:

**Reply by coolapso** [reply]
https://github.com/streamdeck-linux-gui/streamdeck-linux-gui/pull/113

should be good on next release

**Reply by oparada1988** [reply]
Thank you for all the hardwork in getting this resolved :-)
